### PR TITLE
Fix bug in ric-orb@v2.0.0

### PR DESCRIPTION
**Pull request by michaelkuechler**
fixes bug introduced in #17 

hardcode main steps, because steps-parameter defaults can not use parameter values themselfs; see also [here](https://discuss.circleci.com/t/cci-2-1-conditional-step-on-list-steps-parameter-type/28432/5)

### CODE REVIEW

File: src/jobs/build_push_image.yml [hunk 1]
```diff
@@ -88,6 +81,9 @@ steps:
         - restore_cache:
             name: restore the dependencies cache
             key: <<parameters.cache_name>>
+  - attach_workspace:
+      at: .
   - steps: << parameters.prebuild_steps >>
-  - steps: << parameters.build_steps >>
+  - build_push:
+      config: << parameters.path >>/isopod.yml
```
Comment (michaelkuechler) [inline src/jobs/build_push_image.yml:88]: now hardcoded!

File: src/jobs/deploy_job.yml [hunk 2]
```diff
@@ -9,69 +9,64 @@ executor:
...
+    default: 'master'
 
 steps:
   - checkout
   - steps: << parameters.predeploy_steps >>
-  - steps: << parameters.deploy_steps >>
+  - deploy:
+      config: << parameters.path >>/isopod.yml
+      to: <<parameters.env>>
+      work_dir: << parameters.work_dir >>
```
Comment (michaelkuechler) [inline src/jobs/deploy_job.yml:69]: now hardcoded

File: src/jobs/build_push_image.yml [hunk 3]
```diff
@@ -88,6 +81,9 @@ steps:
         - restore_cache:
             name: restore the dependencies cache
             key: <<parameters.cache_name>>
+  - attach_workspace:
+      at: .
```
Comment (julien-may) [inline src/jobs/build_push_image.yml:85]: i was not aware that this was missing, meaning this job was quite unuseful if you were building the application within the docker build process. awesome you added it!

may i ask you to document this fact?

**Review by julien-may — APPROVED** [review]
well done!

i am a bit confused why you removed the quotes from the `.` in the `path` parameter. at the end the default value is still a string and therefore should be quoted as such.

however... this is just a detail and i am aware that this works regardless.

File: src/jobs/build_push_image.yml (hunk 3 above)
Comment (michaelkuechler) [reply src/jobs/build_push_image.yml:85]: yea it kinda does not matter
im honestly not 100% sure whats the pros and cons for no-quotes, single-quotes, and double-quotes. though the latter AFAIK converts escape sequences like `\n` whereas single quote does not.

TLDR: not sure which style we prefer. if we agree on something im happy to follow

**Review by robert-nemet — APPROVED** [review]

File: src/jobs/build_push_image.yml (hunk 3 above)
Comment (michaelkuechler) [reply src/jobs/build_push_image.yml:85]: > i am a bit confused why you removed the quotes from the . in the path parameter. at the end the default value is still a string and therefore should be quoted as such.

yea it kinda does not matter
im honestly not 100% sure whats the pros and cons for no-quotes, single-quotes, and double-quotes. though the latter AFAIK converts escape sequences like `\n` whereas single quote does not.

TLDR: not sure which style we prefer. if we agree on something im happy to follow

**Comment by michaelkuechler** [comment]
regarding no-/single-/double-quotes:
i dont have a strong opinion nor experience whats best

double quotes does some escaping (e.g. `\n` becomes newline).

if we have a shared practice i gladly apply it. but it didnt seem to be the case from what i saw
